.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.6"
+install:
+  - pip3 install -r requirements.txt
+script:
+  # Unit tests
+  - python3 -m topic_dynamics.parsers
+  - python3 -m unittest discover topic_dynamics/tests -t .
```

Comment:
let's keep a new line at the end of the file. I think `-t` flag is not required here - does it work without it?

Reply:
Doesn't work without it.